make/operator.example.mk.help.log

```diff
@@ -13,6 +13,7 @@ image-ocp-openshift-apiserver-operator
 images
 telepresence
 test
+test-operator-integration
```

Comment:
maybe we should indicate that the new target is intended the mom operator by renaming to `test-operator-mom-integration` ?

Reply:
I can go either way. I picked this one mostly because it already exists and we have a presubmit for it in openshift/multi-operator-manager.